api/grpcserver/grpcserver_test.go

```diff
@@ -4,6 +4,7 @@ package grpcserver
 import (
 	"bytes"
 	"context"
+	"crypto/ed25519"
```

Comment:
why sometimes this and sometimes "github.com/spacemeshos/ed25519-recovery"

Reply:
`github.com/spacemeshos/ed25519-recovery` uses aliases for types like `PrivateKey`, `PublicKey`, etc.

I only left `github.com/spacemeshos/ed25519-recovery` where `ExtractPublicKey`, `Verify` or `Sign` from our library are used, since everything else is the same as the official package from the standard library.

Reply:
this is confusing. whats the downside of using the same import?

Reply:
none they can be used interchangeably.

I reverted where it was possible.